com.budgetwithbuckets.Buckets.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=Buckets
+Exec=com.budgetwithbuckets.Buckets
```

Comment:
Should be buckets.sh I think

Reply:
to be clearer: this should match what's in `command` in the manifest (maybe with arguments)